config/config_as_cty.go

```diff
@@ -25,7 +25,7 @@ func terragruntConfigAsCty(config *TerragruntConfig) (cty.Value, error) {
 	output["terragrunt_version_constraint"] = gostringToCty(config.TerragruntVersionConstraint)
 	output["download_dir"] = gostringToCty(config.DownloadDir)
 	output["iam_role"] = gostringToCty(config.IamRole)
-	output["prevent_destroy"] = goboolToCty(config.PreventDestroy)
+	output["prevent_destroy"] = goboolToCty(*config.PreventDestroy)
```

Comment:
Does this need a check that it's not `nil`?

Reply:
I guess so, thank you for point that out.